addons/pos_loyalty/static/tests/tours/pos_loyalty_loyalty_program_tour.js

```diff
@@ -43,9 +43,8 @@ registry.category("web_tour.tours").add("PosLoyaltyLoyaltyProgram1", {
             PosLoyalty.orderTotalIs("12.80"),
             ProductScreen.clickPartnerButton(),
             ProductScreen.clickCustomer("AAA Test Partner"),
-            PosLoyalty.isRewardButtonHighlighted(true),
```

Comment:
You can remove this check. Since the order has 4 items of Whiteboard Pen, then, the cashier should be notified that there is a potential reward on the order, thus, the reward button should be highlighted. Clicking the reward button should automatically add the free product.